diameter-stack/build.gradle

```diff
@@ -2,8 +2,77 @@ plugins {
   id "org.jetbrains.kotlin.jvm" version "1.2.30"
   id "idea"
   id "java-library"
+  id "signing"
+  id "maven"
 }
 
+version="1.0.0-SNAPSHOT"
+
+task javadocJar(type: Jar) {
+    classifier = 'javadoc'
+    from javadoc
+}
+
+task sourcesJar(type: Jar) {
+    classifier = 'sources'
+    from sourceSets.main.allSource
+}
+
+artifacts {
+    archives javadocJar, sourcesJar
+}
+
+signing {
+    sign configurations.archives
+}
+
+
+uploadArchives {
+  repositories {
+    mavenDeployer {
+      beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
+
+      repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
+        authentication(userName: ossrhUsername, password: ossrhPassword)
+      }
+
+      snapshotRepository(url: "https://oss.sonatype.org/content/repositories/snapshots/") {
+        authentication(userName: ossrhUsername, password: ossrhPassword)
+      }
+
+      pom.project {
+        name 'Example Application'
+        packaging 'jar'
+        // optionally artifactId can be defined here
+        description 'A library to communicate over the Diameter protocol.'
+        url 'http://www.example.com/example-application'
```

Comment:
Should we set this URL to 'https://github.com/ostelco/ostelco-core/tree/develop/diameter-stack'

Reply:
I have changed this.